### PR TITLE
code and xmldocs typo fixes

### PR DESCRIPTION
**Pull request by InFTord**
# Summary
Fixes multiple typos in XMLDocs.

# Changes proposed
* multiple adminstrator typo fixes

# Notes
there is a typo in event name (?) possible breaking change?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -398,4 +398,7 @@ FodyWeavers.xsd
 *.sln.iml
 
 # Credentials for the DSharpPlus.Test bot
-DSharpPlus.Test/config.json
\ No newline at end of file
+DSharpPlus.Test/config.json
+
+# Github Codespcaes configuration file
```
Comment (IDoEverything) [inline .gitignore:403]: typo

File: DSharpPlus/Enums/Message/MessageType.cs [hunk 2]
```diff
@@ -99,7 +99,7 @@ public enum MessageType : int
         GuildDiscoveryDisqualified = 14,
 
         /// <summary>
-        /// Message indicating a guild was readded to guild discovery.
+        /// Message indicating a guild was readed to guild discovery.
```
Comment (IDoEverything) [inline DSharpPlus/Enums/Message/MessageType.cs:102]: This was correct before. Maybe change it to `re-added` for clarity.

**Review by IDoEverything — CHANGES_REQUESTED** [review]

File: DSharpPlus/Enums/Message/MessageType.cs (hunk 2 above)
Comment (InFTord) [reply DSharpPlus/Enums/Message/MessageType.cs:102]: sure!

**Comment by InFTord** [comment]
git i didn't wanted to push everything that i did :bruh:

**Comment by InFTord** [comment]
will continue tomorrow, thanks @IDoEverything for reviewing my changes!

**Comment by InFTord** [comment]
im done, feel free to find more typos that i missed/typos that i accidentally did

**Comment by InFTord** [comment]
ready to merge btw, as i said im done with this PR

**Review by Naamloos — APPROVED** [review]
Nothing public facing has been changed (except for one event name but I doubt it will bring breaking changes).

I feel confident enough merging this.

**Pull request merged**